src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -114,6 +117,16 @@ public void hoodMotorStop() {
         hoodMotor.set(STOP_MOTOR);
     }
 
+    /**
+     * Sets the velocity of both flywheel motors
+     * 
+     * @param velocity
+     *            the velocity of the flywheel motors
+     */
+    public void setFlywheelVelocity(double velocity) {
+        flywheelMotor1.set(ControlMode.Velocity, velocity);
```

Comment:
Both flywheel motors need to be at the same speed?

Reply:
In `configMotors()`, `flywheelMotor2` is set to follow `flywheelMotor1`, which will make both of them have the same speed.

Reply:
Roger